homeassistant/components/zha/light.py

```diff
@@ -223,15 +230,13 @@ def async_restore_last_state(self, last_state):
             self._state = bool(level)
             if level:
                 self._brightness = level
-
-        if brightness is None or brightness:
+        else:
```

Comment:
No, the purpose of this condition was always to call ON, so GE dimmers work as well.

Reply:
Doesn't move_to_level_with_on_off work on GE dimmers?

Reply:
It does but it sticks to the specs way to close which is causing issues. Per specs: `Before commencing any command that has the effect of increasing CurrentLevel, the OnOff attribute of the On/Off cluster on the same endpoint, if implemented, SHALL be set to On` The keyword here being `increasing`. So if the light was turned of at level 128 and you send to a GE Dimmer `move_to_level_with_on_off` to level 128, then it is not increasing and therefore does not turn the light on.

Reply:
Oh great! :D